mdetsims/sim_utils.py

```diff
@@ -30,8 +30,12 @@ class Sim(object):
 
     Parameters
     ----------
+    fraction : float, 0 to 1
```

Comment:
We should make the name of this parameter more descriptive. Maybe `frac_ex`?

Reply:
Let's switch the default to be 0 and change the logic below so that this paraneter is the fraction of objects with the extra shear.

Reply:
Sure!